### ISSUE TITLE
Add translations for albanian

### ISSUE THREAD

**Issue by gerardkraja** [squidfunk/mkdocs-material#7564]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "al",
  "direction": "ltr",
  "action.edit": "Ndrysho këtë faqe",
  "action.skip": "Kalo te përmbajtja",
  "action.view": "Shiko burimin e kësaj faqeje",
  "announce.dismiss": "Mos e shfaq më këtë",
  "blog.archive": "Arkiva",
  "blog.categories": "Kategoritë",
  "blog.categories.in": "në",
  "blog.continue": "Vazhdo leximin",
  "blog.draft": "Draft",
  "blog.index": "Kthehu te indeksi",
  "blog.meta": "Të dhëna",
  "blog.references": "Linqe të afërta",
  "clipboard.copy": "Kopjo në clipboard",
  "clipboard.copied": "U kopjua në clipboard",
  "consent.accept": "Prano",
  "consent.manage": "Administro parametrat",
  "consent.reject": "Refuzo",
  "footer": "Fundi i faqes",
  "footer.next": "Tjetër",
  "footer.previous": "E mëparshme",
  "header": "Koka",
  "meta.comments": "Komente",
  "meta.source": "Burimi",
  "nav": "Navigacion",
  "readtime.one": "Lexim 1 min",
  "readtime.other": "Lexim # min",
  "rss.created": "Burim RSS",
  "rss.updated": "Burim RSS me përmbajtje të përditësuar",
  "search": "Kërko",
  "search.placeholder": "Vendmbajtës",
  "search.share": "Shpërndaj",
  "search.reset": "Zbraz",
  "search.result.initializer": "Duke filluar kërkimin",
  "search.result.placeholder": "Shkruaj për të filluar kërkimin",
  "search.result.none": "Nuk ka dokumente të përputhshme",
  "search.result.one": "1 dokument i përputhshëm",
  "search.result.other": "# dokumente të përputhshme",
  "search.result.more.one": "1 më shumë në këtë faqe",
  "search.result.more.other": "# më shumë në këtë faqe",
  "search.result.term.missing": "Mungon",
  "select.language": "Zgjidh gjuhën",
  "select.version": "Zgjidh versionin",
  "source": "Shko te burimi",
  "source.file.contributors": "Bashkëpunues",
  "source.file.date.created": "Krijimi",
  "source.file.date.updated": "Përditësimi i fundit",
  "tabs": "Skedat",
  "toc": "Tabela e përmbajtjes",
  "top": "Kthehu në krye"
}[key] }}{% endmacro %}
```

### Country flag

:flag_al:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 74ca2341a.

**Comment by squidfunk** [comment]
Released as part of [9.5.38](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.38).
